dist/APIs/controllers/AUTH/authentication.js

```diff
@@ -0,0 +1,188 @@
+"use strict";
```

Comment:
Add dist folder to gitignore...It's very important

Reply:
I already did but it is not ignoring.